README.md

```diff
@@ -208,6 +208,32 @@ const getMarkdown = () => {
 - *Ruby* - [`<<~` Heredocs](https://www.rubyguides.com/2018/11/ruby-heredoc/)
 - *Swift* - [multiline string literals](https://github.com/mmkal/proposal-multi-backtick-templates) using triple-quotes - strips margin based on whitespace before closing delimiter
 
+## Syntax Alternatives Considered
+
+Some potential alternatives to the multi-backtick syntax:
+
+- A built-in runtime method along the lines of the `dedent` library, e.g. `String.dedent`:
+  - Pros:
+    - Easier to implement
+    - No new syntax
+  - Cons:
+    - Non-zero runtime impact (though less than a js library)
+    - Unclear how/whether it could work with tagged template literals
+    - More verbose
+    - Lends itself less well to ecmascript-compatible data formats like json5 and forks aiming at modern es features like [json6](https://github.com/d3x0r/JSON6), [jsox](https://github.com/d3x0r/JSOX), [jsonext](https://github.com/jordanbtucker/jsonext) and [ESON](https://github.com/mmkal/eson)
```

Comment:
Can you explain? I think it'd work fine.

Reply:
Json5 and these derivatives have a design goal of being a ecmascript-compatible data formats, as more human-friendly alternatives to json (single- or double-quotes, unquoted object properties, trailing commas etc.). So ecmascript "static" syntax fits, but function calls don't really. Similarly, json5 doesn't support `.trim()` right now.